.doctrine-project.json

```diff
@@ -5,11 +5,17 @@
     "docsSlug": "doctrine-persistence",
     "versions": [
         {
-            "name": "2.0",
-            "branchName": "master",
+            "name": "3.0",
+            "branchName": "3.0.x",
```

Comment:
Shouldn't this be `master` since you're changing the branch-alias for master to 3.0.x?

Reply:
Absolutely yeah :sweat_smile: